snippets/javascript/nuxt/nuxt.json

```diff
@@ -0,0 +1,87 @@
+{
+  "nuxt-nuxtlink": {
+    "prefix": ["nuxt nuxtlink", "nuxtlink"],
+    "body": ["<nuxt-link $1>$2</nuxt-link>$0"],
+    "description": "nuxt-link element",
```

Comment:
Let's also remove the description attribute for snippets where the description duplicates the snippet content. Description could be used for the purpose of adding extra clarification/definition of the snippet, maybe some optional parameters and their types. I guess good example is 
https://github.com/snipsnapdev/snipsnap/blob/master/snippets/javascript/react-intersection-observer/react-intersection-observer.json
https://github.com/snipsnapdev/snipsnap/tree/master/snippets/javascript/lodash

Reply:
Okay.